backend/controllers/retentionReasons.js

```diff
@@ -1,24 +1,27 @@
+/* eslint-disable consistent-return */
 const qs = require('qs')
 const {
   app: { notmEndpointUrl: dpsUrl },
 } = require('../config')
 const { serviceUnavailableMessage } = require('../common-messages')
+const { formatTimestampToDate, formatTimestampToDateTime } = require('../utils')
 
 const retentionReasonsFactory = (elite2Api, dataComplianceApi, logError) => {
   const sortReasonsByDisplayOrder = reasons => reasons.sort((r1, r2) => (r1.displayOrder > r2.displayOrder ? 1 : -1))
 
-  const getAgencyDescription = (agencies, agencyId) => {
-    const agency = agencies.find(a => a.agencyId === agencyId)
-    return agency ? agency.description : 'Unknown'
-  }
-
   const getOffenderUrl = offenderNo => `${dpsUrl}offenders/${offenderNo}`
 
+  const formatDate = date => date && formatTimestampToDate(date)
```

Comment:
these format methods handle null anyway?

Reply:
e.g: https://github.com/ministryofjustice/prisonstaffhub/pull/882/files#diff-d2bd71a8ef7316942b1e5a4166909718L140

Reply:
Ah right you are.  For some reason I'd convinced myself that they wouldn't handle it!